### DISCUSSION TITLE
Unable to search after tapping `homebrew/cask-versions`

### DISCUSSION THREAD

**Discussion by esgott** [Homebrew/discussions#3944] [Casks] [answered]
After running `brew tap homebrew/cask-versions` when I try to search:
```
$ brew search jdk
Error: Cask adoptopenjdk8 exists in multiple taps:
  homebrew/cask-versions/adoptopenjdk8
  adoptopenjdk/openjdk/adoptopenjdk8
```

If I try to untap, similar thing happens
```
$ brew untap homebrew/cask-versions
Error: Refusing to untap homebrew/cask-versions because it contains the following installed formulae or casks:
adoptopenjdk8
```

Also, I can't uninstall this package:
```
$ brew uninstall adoptopenjdk8
Error: Cask adoptopenjdk8 exists in multiple taps:
  homebrew/cask-versions/adoptopenjdk8
  adoptopenjdk/openjdk/adoptopenjdk8
```

It feels like catch 22, can you help me how to get back to a working state?
Also, this feels like a bug to me, uninstall shouldn't fail because the package exists in multi taps.

**Comment by SMillerDev** [comment, marked as answer]
The uninstall is not a bug, you just have to use the full name of the package. Search and untap seems like a bug.
